pjsip/src/pjsua2/endpoint.cpp

```diff
@@ -2515,9 +2517,10 @@ void Endpoint::setCodecOpusConfig(const CodecOpusConfig &opus_cfg)
 
    PJSUA2_CHECK_EXPR(pjmedia_codec_opus_set_default_param(&new_opus_cfg,
                                                           &param));
-}
-
+#else
+    PJ_UNUSED_ARG(opus_cfg);
 #endif
```

Comment:
What if it throws some exception (e.g: `PJ_ENOTSUP`) when Opus is not enabled? (also in `getCodecOpusConfig()`)